src/wolfgame.py

```diff
@@ -6994,13 +7014,16 @@ def vote_gamemode(var, wrapper, gamemode, doreply):
         return
 
     if gamemode not in var.GAME_MODES.keys():
-        match, _ = complete_match(gamemode, var.GAME_MODES.keys() - ["roles", "villagergame"] - var.DISABLED_GAMEMODES)
-        if not match:
-            if doreply:
-                wrapper.pm(messages["invalid_mode_no_list"].format(gamemode))
+        matches = complete_match(gamemode, var.GAME_MODES.keys() - {"roles", "villagergame"} - var.DISABLED_GAMEMODES)
+        if not matches:
+            wrapper.pm(messages["invalid_mode_no_list"].format(gamemode))
             return
-        gamemode = match
-
+        if len(matches) > 1:
+            wrapper.pm(messages["invalid_mode"].format(gamemode, ", ".join(matches)))
+            return
+        else:
```

Comment:
The `else` is redundant here because of the `return`, just put the next line outside of the `if` block unindented. (In some cases `if`/`else` still makes for cleaner code even if return is used, but this is not one of those cases.)